### PR TITLE
About description + Guide&Team links + help tooltips

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
This PR adds 
- a description, guide link, and team link to the about dialog
- help button to the operations group
- help tooltips activated by the help button
- 
@pablo-mayrgundter this is the same request you approved on Wednesday.

### CODE REVIEW

File: src/Components/OpenModelControl.jsx [hunk 1]
```diff
@@ -37,9 +37,9 @@ export default function OpenModelControl({fileOpen}) {
           title={'Open IFC'}
           onClick={() => setIsDialogDisplayed(true)}
           icon={<OpenIcon/>}
-          placement={'top'}
+          placement={'right'}
           selected={isDialogDisplayed}
-          dataTestId='open-ifc'
+          dataTestId='open-ifc'ß
```
Comment (pablo-mayrgundter) [inline src/Components/OpenModelControl.jsx:42]: typo

File: src/Components/OperationsGroup.jsx [hunk 2]
```diff
@@ -104,11 +113,18 @@ export default function OperationsGroup({deselectItems}) {
            title='Notes'
            icon={<NotesIcon/>}
            selected={isNotesOn}
-           onClick={() => toggle('Notes')}
+           onClick={() => {
+             turnOffTooltips()
+             toggle('Notes')
+           }
```
[the remark this answers is not in the document]
Comment (OlegMoshkovich) [reply src/Components/OperationsGroup.jsx:119]: i think fixed.

File: src/Containers/CadView.jsx [hunk 3]
```diff
@@ -644,7 +644,7 @@ function OperationsGroupAndDrawer({deselectItems}) {
         <Box
           sx={{
             position: 'absolute',
-            top: 0,
+            top: 3,
```
Comment (pablo-mayrgundter) [inline src/Containers/CadView.jsx:647]: Please add a comment:

`TODO(pablo): CadView just has two sub-components the left and right group, and their first elements should be same height and offset so they line up naturally.. this is a shim for the misalignment you see with tooltips without it`

**Review by pablo-mayrgundter — APPROVED** [review]
Just a few nits

**Pull request merged**